Preparation twig transition (#914)
contao/docs: docs/manual/layout/templates/template-inheritance.en.md

Contao template inheritance allows overwriting only certain sections of a template (blocks).

### Adjust blocks

Many templates already structure their contents. Only contents wrapped in such blocks can be adjusted.
First, the base template must be declared. Then you can provide new block content.

#### PHP Example
The `fe_page.html` template contains multiple blocks (such as `head`, `meta`, `body`, `footer`). If we only want to add
another meta tag, we could write the following:

> ```php
> // /templates/fe_page.html5
> 

Reviewer: ```suggestion
```html
<!-- templates/fe_page.html5 -->
```

Contao template inheritance allows overwriting only certain sections of a template (blocks).

### Adjust blocks

> Many templates already structure their contents. Only contents wrapped in such blocks can be adjusted.

Reviewer 2: This might suggest, that you cannot overwrite templates, which is an alternative to extending.

Author: Habe hierbei lediglich die bestehenden Inhalte übernommen:

https://docs.contao.org/manual/de/layout/templates/vererbung/ oder auch
https://docs.contao.org/manual/en/layout/templates/template-inheritance/

The original block content is available via `$this->parent()`.

```html
<!-- templates/fe_page.html5 -->
<?php $this->extend('fe_page'); ?>

<?php $this->block('meta'); ?>
  <?php $this->parent(); ?>

  <meta name="author" content="John Doe">
<?php $this->endblock(); ?>
```

{{% /tab %}}
{{% tab name="Twig" %}}

> {{< version "4.13" >}}

Reviewer 3: ```suggestion
{{< version "4.12" >}}
```

> First, the base template must be declared with `{% extends 'name-of-the-template' %}`. Then you can provide new block 

Reviewer 2: ```suggestion
First, the base template must be declared with `{% extends '@Contao/name-of-the-template' %}`. Then you can provide new block 
```
